drag_and_drop_v2/templates/html/js_templates.html

```diff
@@ -82,45 +82,58 @@
     </div>
 </script>
 
-<script id="zone-dropdown-tpl" type="text/html">
-    <option value="{{ uid }}" {{ selected }}>{{ title }}</option>
+<script id="zone-checkbox-tpl" type="text/html">
+    <div class="row">
+        <label>
+            <input type="checkbox"
+                   value="{{ zoneUid }}"
+                   class="zone-checkbox"
+                   {{ checked }} />
+            {{ title }}
+        </label>
+    </div>
 </script>
 
 <script id="item-input-tpl" type="text/html">
     <div class="item">
         <div class="row">
-            <label for="item-{{id}}-text">{{i18n "Text"}}</label>
+            <label for="item-{{id}}-text" class="h3 zone-checkbox-label">{{i18n "Text"}}</label>
```

Comment:
@haikuginger Probably just a typo -- if we need an additional class to style this, it would make sense to call it `zone-text-label` I think.